APCAppCore/APCAppCore/Startup/APCAppDelegate.h

```diff
@@ -60,6 +60,7 @@ static NSString*    const kDatabaseName                     = @"db.sqlite";
 @property (strong, nonatomic) APCProfileViewController * profileViewController;
 @property (nonatomic) BOOL disableSignatureInConsent;
 @property (nonatomic, strong) APCDataUploader *dataUploader;
+@property (nonatomic) BOOL showShareAppInOnboarding;
```

Comment:
I realize you probably started this before the onboarding manager was merged. This setting should be a property of APCOnboardingManager as to not re-introduce dependencies on the app delegate in `APCInEligibleViewController`.